.project

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
I don't care if we add this in, more just curious what this is 😂

Reply:
lmao no idea why I made this change